### PR TITLE
Misc queue from 2018 (sorry, lost sources) 

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -8617,6 +8624,9 @@ hunderd->hundred
 hundreths->hundredths
 hundrets->hundreds
 hunrgy->hungry
+huricane->hurricane
+huristic->heuristic
+hurse->hearse
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Suggest nurse

Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -8551,28 +8551,34 @@ hirarcies->hierarchies
 hirearcy->hierarchy
 hismelf->himself
 hisory->history
+hist->heist
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Suggest his

Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

**Pull request merged**